demo/src/main/java/io/monstarlab/mosaic/features/SliderDemo.kt

```diff
@@ -0,0 +1,210 @@
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Switch
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableFloatStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.rotate
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import io.monstarlab.mosaic.slider.Slider
+import io.monstarlab.mosaic.slider.SliderColors
+import io.monstarlab.mosaic.slider.SliderValueDistribution
+import io.monstarlab.mosaic.slider.rememberSliderState
+import kotlin.math.roundToInt
+import androidx.compose.material3.Slider as MaterialSlider
+
+@Composable
+fun SliderDemo() = Scaffold(modifier = Modifier) {
+    var materialSliderValue by remember { mutableFloatStateOf(50f) }
+
+    Column(
+        modifier = Modifier
+            .padding(it)
+            .padding(16.dp),
+        verticalArrangement = Arrangement.spacedBy(32.dp),
+    ) {
+        MaterialSlider(
+            value = materialSliderValue,
+            onValueChange = { materialSliderValue = it },
+            valueRange = 0f..100f,
+        )
+
+        MosaicSliderDemo()
+    }
+}
+
+@Composable
+fun MosaicSliderDemo() {
+    Column {
+        val colors = SliderColors(
+            activeTrackColor = Color.Black,
+            disabledRangeTrackColor = Color.Red,
+            thumbColor = Color.Yellow,
+        )
+        var enabled by remember { mutableStateOf(true) }
+        var isCustom by remember { mutableStateOf(false) }
+        var linearDistribution by remember { mutableStateOf(false) }
+
+        val modifier = if (isCustom) {
+            Modifier
+                .clip(RoundedCornerShape(16.dp))
+                .height(15.dp)
+        } else {
+            Modifier
+        }
+
+        val parabolic: SliderValueDistribution = remember {
+            SliderValueDistribution.parabolic(
+                a = (1000 - 100 * 0.1f) / (1000 * 1000),
```

Comment:
Bunch of magic numbers, can we add a comment or move them to constants in the `companion object`?

Reply:
Perfect, just for the people who'll be looking at the demo to understand what's that about